### PR TITLE
Localalerts settings

### PR DESCRIPTION
**Pull request by AdrianLxM**

### CODE REVIEW

**Comment by AdrianLxM** [comment]
* Don't see pump as initialized if `maxBasal == 0`
* Guard Dialogs from multiple clicks
* Map Overview-Notifications Android-Notifications (optional); remove direct wear Notifications.
* Stale data alert and pump not reachable alert. Based on https://github.com/MilosKozak/AndroidAPS/pull/492 from @jotomo 
  * Adjustable in settings
  * Not active when disconnected

File: app/src/main/java/info/nightscout/androidaps/plugins/Overview/notifications/NotificationStore.java [hunk 1]
```diff
@@ -49,21 +54,53 @@ public void add(Notification n) {
...
+                Intent alarm = new Intent(MainApp.instance().getApplicationContext(), AlarmSoundService.class);
+                alarm.putExtra("soundid", n.soundId);
+                MainApp.instance().startService(alarm);
+            }
+
+            //Only pipe through to wear if no system notification is raised (should show on wear anyways)
+            /*WearPlugin wearPlugin = MainApp.getSpecificPlugin(WearPlugin.class);
+            if(wearPlugin!= null && wearPlugin.isEnabled()) {
+                wearPlugin.overviewNotification(n.id, "OverviewNotification:\n" + n.text);
+            }*/
```
Comment (jotomo) [inline app/src/main/java/info/nightscout/androidaps/plugins/Overview/notifications/NotificationStore.java]: @AdrianLxM is the commented-out code still useful?

**Comment by AdrianLxM** [comment]
Did the code cleanup as @jotomo suggested.

**Comment by jotomo** [comment]
👍 Tested, LGTM

**Comment by MilosKozak** [comment]
replaced by https://github.com/MilosKozak/AndroidAPS/commit/d1aec1ff18af9fd28b9f435c611982edd349ed4a
sorry again
@AdrianLxM please quick review

**Comment by AdrianLxM** [comment]
Just an idea regarding superbolus:

With the callbacks what could be done now is:
In case of superbolus, call `activeloop.superBolusTo(System.currentTimeMillis() + 2 * 60L * 60 * 1000);` in the callback and only in case of success. Also only trigger the actual bolus from the the callback of setting the 0-temp-basal and only in case of success. (https://github.com/MilosKozak/AndroidAPS/commit/d1aec1ff18af9fd28b9f435c611982edd349ed4a#diff-3efcb22b3b58c755fa15b0261cdbf21bR349)

The standard bolus would then be an `else` branch of the `if (useSuperBolus)`.

**Comment by MilosKozak** [comment]
will you make pr?

**Comment by AdrianLxM** [comment]
The wear-notifications that are no longer needed are still in:
https://github.com/MilosKozak/AndroidAPS/pull/495/files#diff-fed474c0f39bc34a66a905302d4c498eL59

**Comment by AdrianLxM** [comment]
@MilosKozak from reading both diffs, it looks good to me. :) All things I have noticed are enhancements -> opened TODOs for it.
Closing this. Thank you for doing the manual merge!!!

**Pull request closed without merging**